vestapol/web_resources/json_resource.py

```diff
@@ -29,6 +30,7 @@ def __init__(
             self.response_filename,
             self.query_params,
             self.request_headers,
+            self.requested_at
```

Comment:
This will pass the tests, but a developer will get `AttributeErrors` when they subclass this class, unless they know that they need to implement `self.requested_at` (take a look at the [SeptaResource](https://github.com/phillymedia/inq-data-resources/blob/main/catalog/septa.py#L5-L9), for example. This change would break that connector). Since `self.requested_at` powers a core Vestapol functionality and has a sensible default (i.e. "now"), I think it makes sense to ensure that it is set without any developer intervention (ideally, we would set `requested_at` whenever a `request_data()` method is called; there's probably a way to accomplish this with decorators, but that's a story for another day). 

So ultimately we would set `self.requested_at = DateTime.utcnow().replace(microsecond=0)` only once (in `BaseResource.__init__`) and then not have to worry about it again.

That's just my opinion (and I've admittedly flip-flopped on it in the past), open to other approaches as well